Сверстан Header

Alex-nzn opened the pull request:
Добавил пропс titleImg

Alex-nzn committed:
Сверстан Header

nobleworkshop commented on src/components/Header/Header.jsx:
```diff
@@ -4,20 +4,25 @@ import Link from "../Link/Link";
-                <Link link="/podcast" title="Podcast" />
-                <img src={img} alt="line" />
-                <div className={styles.socials}> <Socials /></div>
+        <>
```
Можно поменять fragment на тег header.

nobleworkshop reviewed (changes requested):
Отлично!) 

Компонент Header в ПР из ветки article. Немного не порядок, ну ладно. Должен был быть сделать в отдельной ветке header.

1) Удалить из header.module.css неиспользованные стили, .links  и .links  *.

2) Сделать чтобы компонент Header принимал параметр title (art&life, magazine, podcast ) как опциональный параметр. И если он передан - уже внутри подключал бы нужную svg картинку с названием (сейчас это jpeg картинка которая передается через props).

3) Доработать адаптивность. Как минимум просто перестраивать всё в колонку на мобилках.

Alex-nzn committed:
Сделал title опционально

удалил неиспользуемые классы
доработал адаптивность
заменил fragment на тег header

Commit:
Пофиксил заголовки для секций в Header
